dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/NetUtils.java

```diff
@@ -130,7 +128,7 @@ private static synchronized InetAddress getLocalAddress0() {
                     Optional<InetAddress> addressOp = toValidAddress(addresses.nextElement());
                     if (addressOp.isPresent()) {
                         try {
-                            if (addressOp.get().isReachable(100)) {
+                            if (addressOp.get().isReachable(300)) {
```

Comment:
100 is actually better. Only in the windows environment will there be a greater delay. 
If the user environment really needs it, it is recommended to modify it yourself.

Reply:
I changed it to 200. I have tested several times before that in some cases, 100 will time out but 200 will not

Reply:
There is a scenario: the 2nd ip is used, not the 1st ip
![image](https://user-images.githubusercontent.com/4902714/118223277-92cd3600-b4b3-11eb-920c-46c3bfe1cc2f.png)
![image](https://user-images.githubusercontent.com/4902714/118223548-230b7b00-b4b4-11eb-8775-57abc66abf41.png)